src/OpenLoco/Windows/ScenarioSelect.cpp

```diff
@@ -341,8 +341,10 @@ namespace OpenLoco::Ui::Windows::ScenarioSelect
                 continue;
 
             if (y + rowHeight < dpi->y || y > dpi->y + dpi->height)
+            {
+                y += rowHeight;
                 continue;
-
+            }
```

Comment:
Please leave the whitespace after the closing brace.

```suggestion
            }

```